study.md

```diff
@@ -43,7 +45,7 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+I'm not sure, but I believe it is because an enumerable is in a sense defined by its length. That is to say, an enumberable will always be a specificed set of numbers, whereas an array can be and arbitrary list of things, which may increase of decrease due to invocation of other methods.
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.